packages/playground/package.json

```diff
@@ -48,6 +49,7 @@
     "@rjsf/fluentui-rc": "^5.15.1",
     "@rjsf/material-ui": "^5.15.1",
     "@rjsf/mui": "^5.15.1",
+    "@rjsf/mui-joy": "file:../mui-joy",
```

Comment:
This will need to be replace in a follow-on PR after merging

Reply:
can i push in the @rjsf namespace?